project_timeline_hr_timesheet/views/project_task_view.xml

```diff
@@ -14,7 +14,13 @@
             <xpath expr="//small[@name='planned_hours']" position="after">
                 <div class="o_project_timeline_hr_timesheet_progress">
                     <div
-                        t-att-class="record.remaining_hours &lt; 0 and 'danger' or ''"
+                        t-if="record.remaining_hours &lt; 0"
```

Comment:
I don't think you need this, just put the other as `t-attf-class`

Reply:
I have tried to make that change but it does not work in this case.
![ejemplo-1](https://user-images.githubusercontent.com/4117568/223176357-4338b0a2-f461-46a1-abaf-e86136c06325.png)

Reply:
It's weird, as the Qweb should do it:

https://www.odoo.com/documentation/14.0/es/developer/reference/javascript/qweb.html

Maybe it's not being rendered properly? Anyway, if not possible, just drop a comment here to know why it's done that way. Please always remember to document each weird thing for not questioning each time why is there.

Reply:
I finally found out how to do it in one line.